tasks/VMware_Windows.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Install VMWare Tools
-  win_chocolatey:
+  chocolatey.chocolatey.win_chocolatey:
```

Comment:
do we also want to update [tasks/virtualbox_Windows.yml](https://github.com/ccdc-opensource/ansible-role-install-vm-tools/blob/5e53e2fe0d0e851107a227c02d6a8a604623fc93/tasks/virtualbox_Windows.yml#L3) to be the same?

Reply:
Already done! :)